libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationViewApi.kt

```diff
@@ -105,4 +107,17 @@ class NavigationViewApi internal constructor(
     fun isReplayEnabled(): Boolean {
         return store.state.value.tripSession.isReplayEnabled
     }
+
+    /**
+     * Get access to [MapView] instance created by [NavigationView] synchronously. This API has no
+     * knowledge of previous [MapView] instance in case [MapView] is changed at runtime. Hence it
+     * is easy to create dangling subscriptions after a [MapView] update.
+     *
+     * It's recommended to also observe [NavigationViewListener.onMapViewAttached] and
+     * [NavigationViewListener.onMapViewDetached] as they can be reliably used to remove
+     * subscriptions whenever a new instance of [MapView] is available
+     */
+    fun getMapView(): MapView? {
```

Comment:
I think we should just remove this, to prevent people from creating dangling subscriptions.

Reply:
I am going back and forth on this, because we have seen requests in the past from customers asking for synchronous access to few data objects that can otherwise be obtained asynchronously. The only other option is to remove it, but then reintroduce it upon request.

Reply:
Removed.